jobs/upload-dashboards-objects/templates/bin/import-objects

```diff
@@ -23,58 +26,62 @@ if __name__ == "__main__":
     key = '<%= job_dir %>/config/ssl/dashboard-web.key'
 
     logs_url = 'https://<%= p('opensearch_dashboards.host') %>:<%= dashboards_link.p('opensearch_dashboards.port') %>'
+    os.system('cf api <%= api %>')
+    os.system('cf auth "<%= username %>" "<%= password %>"')
+    orgs = os.popen("cf orgs| tail -n +4| tr '\n' ' ' ").read().split(" ")
```

Comment:
So I might be silly, but I worry about making OS level calls from Python because it seems like an opportunity for privilege escalation

A simple solution would be:

- Create a new script: `generate-tenants`
- Have `generate-tenants` that runs these bash commands to populate a file called `org-tenants.txt` (or something) containing the list of tenants
- Update `run` so that `generate-tenants` is called before `import-objects`
- Have this code in `import-objects` read the tenants from `org-tenants.txt` in Python

What do you think?

Also, instead of using username/password auth, let's use client based auth. We can add a new client here: https://github.com/cloud-gov/cg-deploy-cf/blob/main/bosh/opsfiles/clients.yml. This will ensure that our client gets provisioned in each environment

[Then update the `cf auth` command to use client credentials](https://cli.cloudfoundry.org/en-US/v8/auth.html)

Reply:
That could work, we should also update the create tenants script to do the same

Reply:
Should be added in now